src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery.Test/Commands.RecoveryServices.SiteRecovery.Test.csproj

```diff
@@ -34,12 +34,31 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Hyak.Common, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
Do not add duplicate references. Most of these added references exist in our Common.Dependences.targets or Common.Dependencies.Test.targets file. In Visual Studio in the Solution Explorer, you'll see several references duplicated, one resolved properly and one with a yellow exclamation mark. Please remove the duplicate references.